cluster-autoscaler/FAQ.md

```diff
@@ -792,6 +792,7 @@ The following startup parameters are supported for cluster autoscaler:
 | `cordon-node-before-terminating` | Should CA cordon nodes before terminating during downscale process | false
 | `record-duplicated-events` | Enable the autoscaler to print duplicated events within a 5 minute window. | false
 | `debugging-snapshot-enabled` | Whether the debugging snapshot of cluster autoscaler feature is enabled. | false
+| `allow-scale-down-on-custom-controller-owned-pods` | Don't block node scale-down if a pod owned by a custom controller is running on the node. | false
```

Comment:
This is a different flag name to the on in main. Please update.

Reply:
Looks like I missed this. Thank you for noticing.

Reply:
Updated.